### ISSUE TITLE
Please make this opt-in

### ISSUE THREAD

**Issue by edent** [Sujal-7/WikiSummarizerBot#1]
Don't see why it is up to me to manually opt out of this.

It leads to this situation where multiple unwanted bots reply to a post.

![Screenshot 2021-06-24 at 09-32-31 r LegalAdviceUK - Comment by u WikiSummarizerBot on ”What services can be offered under h](https://user-images.githubusercontent.com/837136/123230435-280a1400-d4cf-11eb-94af-60c72512b5da.png)

**Comment by jobbautista9** [comment]
I agree. I find this bot useless. What's the point of hyperlinking then, if you're going to just paste an overview of the article anyway? If a subreddit find this bot useful, then let them opt-in. But please don't add noise into subreddits that haven't explicitly approved your bot.

**Comment by willhughes-au** [comment]
The world does not need yet another Wiki bot that's opt-out.
